specification/resources/resource-manager/Microsoft.Authorization/stable/2019-01-01/policyAssignments.json

```diff
@@ -0,0 +1,703 @@
+{
```

Comment:
I dont see any delta from 2018-05-01 api version. Is that correct?

Reply:
Yes, no change.